src/components/vm/overview/cpuModal.jsx

```diff
@@ -229,7 +229,7 @@ export const CPUModal = ({ vm, maxVcpu, models }) => {
                            <Popover bodyContent={maxVcpu
                                ? cockpit.format(_("Maximum number of virtual CPUs allocated for the guest OS, which must be between 1 and $0"), parseInt(maxVcpu))
                                : _("Maximum number of virtual CPUs allocated for the guest OS")}>
-                               <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                               <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230629-150027-4152f662-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)